### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Winner95**
Add `getComponentWrapper` function to config to wrap the content of sandboxes, for example to use special context providers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,16 @@ module.exports = {
         return path.indexOf(`${componentMeta.fileNameWithoutPrefix}.spec`) !== -1;
     },
 
+    /**
+     * This function will wrap the content of sandboxes, 
+     * for example to use special context providers within provided
+     * Class or Component
+     * @returns Object
```
Comment (kdzuin) [inline README.md]: not sure, if it returns object

File: README.md [hunk 2]
```diff
@@ -71,6 +71,16 @@ module.exports = {
         return path.indexOf(`${componentMeta.fileNameWithoutPrefix}.spec`) !== -1;
     },
 
+    /**
+     * This function will wrap the content of sandboxes, 
+     * for example to use special context providers within provided
+     * Class or Component
```
Comment (kdzuin) [inline README.md]: The description is not clear enough and is misleading. Does this function wrap anything on its own?
Can we add something in addition to contexts?

File: README.md [hunk 3]
```diff
@@ -71,6 +71,16 @@ module.exports = {
...
+     * @returns Object
+     */
+    getComponentWrapper() {
+        require('MyComponent');
```
Comment (kdzuin) [inline README.md]: This function doesn't return anything

Comment (Winner95) [reply README.md]: missprint, fixed

**Pull request merged**